site/docs/developer_documentation/ONV_path_manipulation.md

```diff
@@ -0,0 +1,212 @@
+
+![init_path](/GQCP/img/ONVPath_10110.png)
+
+This path is the visual representation of ONV `10110`. There are three diagonal arcs (starting at [0, 0], [2, 1] and [3, 2]). These are the coordinates where `annihilate()` can remove an electron. Let's say you `annihilate(0, 0)`. The diagonal arc starting at [0, 0] will become a vertical arc, resulting in a new path:
+
+![firs_annihilate](/GQCP/img/ONVPath_00110_1.png)
+
+The ONV path is now 'open'. To 'close' it, the `create()` function can be used. In this particular case, the path can be closed by creating at coordinate [1, 1], resulting in a new ONV: `01110`.
+
+![firs_create](/GQCP/img/ONVPath_01110.png)
+
+We will come back to this later, when looking at a complete example, side-by-side with the source code. 
+
+Another important function is `leftTranslate()`.
+
+```C++
+/**
+  *  Translate the diagonal arc that starts at the coordinate (p, n) to the left.
+  * 
+  *  @param p        the index of the orbital that should be annihilated
+  *  @param n        the number of electrons in the ONV/path up to the orbital index p
+  */
+ void ONVPath::leftTranslate(const size_t p, const size_t n) {
+
+     // Translating a diagonal arc can be rewritten as an annihilation, followed by a creation.
+     this->annihilate(p, n);
+     this->create(p, n - 1);
+
+     // Since a left-translation describes the process of 'encountering an electron/occupied orbital', the sign factor should be updated according to the fermionic anticommutation rules.
+     this->m_sign *= -1;
+}
+```
+
+In the initial example, the annihilation could be directly followed by a creation, since the next arc was vertical; i.e. an unoccupied orbital. This is not always the case however. If the next orbital is also an occupied orbital, the arc needs to be translated to the left, in order to check whether the subsequent arc represents an occupied or unoccupied orbital. Let us look back at the ONV `10110` after an `annihilate(0, 0)` operation, but assume we don't want to create at coordinate [1, 1], as previously shown. Since our second arc will remain vertical, it can be moved to the left without changing the address. The next index is the starting point of a diagonal arc, which means a creation is not possible there. But since we cannot translate this arc without changing the address, we need `leftTranslate()`. The resulting ONV paths we can be shown as:
+
+![firs_annihilate_shift](/GQCP/img/ONVPath_00110_2_3_side.png)
+
+with the left image showing the situation before `leftTranslate()` and the right image showing the situation after the operation. It is important to note that `leftTranslate()` multiplies the sign of the ONV path. This is done because of the fermionic anti-commutation relation which has to be taken into account each time an occupied orbital is encountered. 
+
+## Example: ONV 10110
```

Comment:
I like your example! It's easy to follow and explains the used functions in a clear way!